File: iOS/Code/Xamarin/bugTrap/bugTrapKit/Trackers/DoneDone/Domain/DoneDoneIssueListItem.cs
using System;

namespace bugTrapKit.DoneDone
{

	public class DoneDoneIssueListItem : DoneDoneIssueBase
	{
		public DateTime LastUpdatedOn { get; set; }

		public DoneDoneSimpleItem LastUpdater { get; set; }

		public DateTime LastViewedOn { get; set; }

		public bool IsPublicIssue { get; set; }
	}
}

File: iOS/Code/Swift/bugTrap/bugTrapKit/Trackers/DoneDone/Domain/DoneDoneIssueListItem.swift
//
//  DoneDoneIssueListItem.swift
//  bugTrap
//
//  Created by Colby L Williams on 8/25/14.
//  Copyright (c) 2014 bugTrap. All rights reserved.
//

import Foundation

class DoneDoneIssueListItem : DoneDoneIssueBase {
	
	var lastUpdatedOn: NSDate?
	
	var lastUpdater: DoneDoneSimpleItem?
	
	var lastViewedOn: NSDate?
	
	var isPublicIssue = false
}